powqutyd/files/src/mqtt.c

```diff
@@ -17,6 +17,9 @@
 #include <stdlib.h>
 #include <errno.h>
 
+#define STATIC_DATA_LENGTH 210
```

Comment:
out of curiosity: where comes the 210 from ?

Reply:
I don't have the paper in front of me atm, but I calculated how many character where possible with in the config file, added the JSON characters and, i think, 2character to be sure i did not leave anything out.
```
-five variables with max length of 32 character
-somewhere around 40 JSON string character
```